manim/mobject/coordinate_systems.py

```diff
@@ -59,6 +59,47 @@
 class CoordinateSystem:
     """
     Abstract class for Axes and NumberPlane
+
+    Examples
+    --------
+
+    .. manim:: CoordSysExample
+        :save_last_frame:
+
+        class CoordSysExample(Scene):
+            def construct(self):
+                # When drawing Axes, the location of the ticks
+                # on the axes depend on x_range and y_range.
```

Comment:
could be shortened and made more concise with:

`# the location of the ticks depends on the x_range and y_range.`